packages/@expo/config-plugins/src/android/EdgeToEdge.ts

```diff
@@ -0,0 +1,136 @@
+import { ExpoConfig } from '@expo/config-types';
```

Comment:
personally i would like to put the plugin into `@expo/prebuild-config`, e.g. `@expo/prebuild-config/plugins/unversioned/react-native-edge-to-edge/withEdgeToEdge.ts`. the main reason is that `@expo/config-plugins` is like a core and would like to keep as minimum as possible. it's not ideal to put react-native-edge-to-edge loader here.